webapp/src/components/App/Singlestudy/explore/Xpansion/Capacities/index.tsx

```diff
@@ -41,6 +38,8 @@ function Capacities() {
     }
   );
 
+  const { data: capacities, reload: reloadCapacities } = res;
```

Comment:
why a second line instead of unstructuring the value in the usePromise response ?

Reply:
I use `res` for UsePromiseCond` component